### PR TITLE
ARROW-2394: [Python] Fixes flake8 issues

### PR DESCRIPTION
**Pull request by AartGoossens**

### CODE REVIEW

File: python/doc/source/conf.py [hunk 1]
```diff
@@ -77,7 +78,6 @@
 # source_suffix = ['.rst', '.md']
 source_suffix = '.rst'
 
-import glob
```
Comment (AartGoossens) [inline python/doc/source/conf.py]: Please double check this one, there might be a reason not to import this earlier.

Comment (pitrou) [reply python/doc/source/conf.py]: I don't think there is. `glob` is a standard library modules, present on all Python installations.

File: python/scripts/test_leak.py [hunk 2]
```diff
@@ -57,4 +57,5 @@ def leak2():
 
         gc.collect()
 
+
```
Comment (pitrou) [inline python/scripts/test_leak.py]: As a sidenote, an idea for a sprint task: investigate what this script does and whether it still exercises a real leak.

File: python/setup.py [hunk 3]
```diff
@@ -291,14 +293,16 @@ def _run_cmake(self):
             shutil.move(pjoin(build_prefix, 'include'),
                         pjoin(build_lib, 'pyarrow'))
 
-            # Move the built C-extension to the place expected by the Python build
+            # Move the built C-extension to the place expected by the Python...
+            # ...build
```
Comment (pitrou) [inline python/setup.py]: The ellipsis doesn't seem ok here.

**Review by pitrou — CHANGES_REQUESTED** [review]
Thanks for doing this. Looks almost right, except for the ellipsis issue I mentioned in the comments.

**Comment by wesm** [comment]
Sorry I missed this patch; was addressed as part of a different PR last month

**Pull request closed without merging**